data/spells/scripts/support/levitate.lua

```diff
@@ -0,0 +1,36 @@
+function onCastSpell(creature, variant)
+	local currentPosition = creature:getPosition()
+	local destinationPosition = creature:getPosition()
+	destinationPosition:getNextPosition(creature:getDirection())
```

Comment:
`getNextPosition` returns the next position instead of changing in place. `local destination = current:getNextPosition(creature:getDirection())` instead.